libcusp/include/galois/graphs/DistributedGraph.h

```diff
@@ -68,6 +68,29 @@ class DistGraph {
 
   using GraphTy = galois::graphs::LC_CSR_Graph<NodeTy, EdgeTy, true>;
 
+  // vector for determining range objects for master nodes + nodes
+  // with edges (which includes masters)
+  //! represents split of all nodes among threads to balance edges
+  std::vector<uint32_t> allNodesRanges;
```

Comment:
I'm assuming that child classes don't actually directly access these. If so, moving them to private is a good idea. Thanks.